app/taskMenu.js

```diff
@@ -21,7 +21,29 @@ function setAutoRun() {
 }
 
 function getUserConnect(callback) {
-  document.getElementById("connect").addEventListener("click", () => { getIP(callback) });
+  document.getElementById("connect").addEventListener("click", () => { showDialog(callback) });
+}
+
+function showDialog(callback) {
```

Comment:
We're not validating the text input here @harshitks2203. Users might not enter a correct URL.

Reply:
ok i will add validation too.